Tensile/SolutionStructs.py

```diff
@@ -3684,6 +3662,163 @@ def assignDerivedParameters(state):
       # disable DTL
       state["DirectToLdsB"] = False
 
+    # LDS padding
```

Comment:
This is a really large code section focused on calculating padding / block size parameters. WOuld it make sense to move this section to its own function?

Reply:
Let me think about making a separate function for this.

Reply:
Updated. I made a separate function for auto ldspad calculation.